### ISSUE TITLE
[Bug]  crash when landing a plane

### ISSUE THREAD

**Issue by srmtyphoone** [jaihysc/Factorio-AircraftRealism#1]
Hi - I just wanted to say thank you for updating this mod, I love the immersive aircraft realism component. I wanted to file a bug report that I had when landing any plane in the mod. I am not sure if this bug was there before or not as I just tried out the mod today. Here is the lua traceback, looks like something happening with the inventory indexing? Thank you for your help and support of the mod.

```
Error while running event AircraftRealism::on_tick (ID 0)
Index out of inventory bounds.
stack traceback:
AircraftRealism/logic/planeTakeoffLanding.lua:8: in function 'insertItems'
AircraftRealism/logic/planeTakeoffLanding.lua:28: in function 'transitionPlane'
AircraftRealism/logic/planeTakeoffLanding.lua:123: in function 'planeLand'
AircraftRealism/logic/planeManager.lua:49: in function 'checkPlanes'
AircraftRealism/control.lua:13: in function <AircraftRealism/control.lua:8>
stack traceback:
[C]: in function 'index'
__AircraftRealism/logic/planeTakeoffLanding.lua:8: in function 'insertItems'
AircraftRealism/logic/planeTakeoffLanding.lua:28: in function 'transitionPlane'
AircraftRealism/logic/planeTakeoffLanding.lua:123: in function 'planeLand'
AircraftRealism/logic/planeManager.lua:49: in function 'checkPlanes'
AircraftRealism/control.lua:13: in function <AircraftRealism/control.lua:8>
```

**Comment by jaihysc** [comment]
Followed up on [Factorio fourms](https://mods.factorio.com/mod/AircraftRealism/discussion/5d7d5a396ef147000b6ef256)

**Comment by jaihysc** [comment]
Fixed with release 0.3.3
